packetbeat/docs/flows.asciidoc

```diff
@@ -0,0 +1,104 @@
+[[flows]]
+== Configuring Flows to Monitor Network Traffic
+
+You can configure Packetbeat to collect and report statistics on network flows.
+A _flow_ is a group of packets sent over the same time period that share
+common properties, such as the same source and destination address and protocol.
+You can use this feature to analyze network traffic over specific protocols on
+your network.
+
+For each flow, Packetbeat reports the number of packets and the total number of
+bytes sent from the source to the destination. Each flow event also contains
+information about the source and destination hosts, such as their IP address and
+location. For bi-directional flows, Packetbeat reports statistics for the reverse
+flow.
+
+Packetbeat collects and reports statistics per flow on the following network
+layers:
```

Comment:
I think it's enough to say it collects info up to the transport layer. The details aren't that important since they can jump over to the exported field docs.